components/post_view/message_attachments/message_attachment/__snapshots__/message_attachment.test.tsx.snap

```diff
@@ -110,7 +111,6 @@ exports[`components/post_view/MessageAttachment should call actions.doPostAction
                 }
               }
               actionExecuting={false}
-              actionExecutingMessage={null}
```

Comment:
These changes are here since we no longer are passing null, but leaving this undefined.